AnkiHubSoftware/ankihub_addon
# [BUILD-917] Update chatbot upsell

## Related issues

https://ankihub.atlassian.net/browse/BUILD-917

## Proposed changes

This changes the chatbot icon if the user is non-premium and removes upsell handling (which is now handled in the web app)

## How to reproduce

- Check in https://github.com/AnkiHubSoftware/ankihub/pull/2492 in the web app.
- Sign in to a non-premium account and go to a deck where the chatbot is enabled.
- Confirm the chatbot icon is changed and open the chatbot to see the new upsell message.

## Screenshots and videos

![image](https://github.com/user-attachments/assets/bea509a0-19d0-4dce-8cad-baa292225275)
![image](https://github.com/user-attachments/assets/11f895f2-8247-4340-ad98-427a8a2ac0f7)
![image](https://github.com/user-attachments/assets/772c03e6-6040-4b8e-86eb-92d624a1ad81)
![image](https://github.com/user-attachments/assets/ff75ce1c-321c-4bf8-bf6d-afa8f285551d)
